src/community/oseo/oseo-core/src/test/resources/postgis.sql

```diff
@@ -206,7 +206,10 @@ create table collection_ogclink (
   "method" varchar,
   "code" varchar,
   "type" varchar,
-  "href" varchar
+  "href" varchar,
```

Comment:
Nope... the ``postgis.sql`` is linked from the documentation as the reference structure for the database. We cannot have testing bits in it.
That said, you can run ``alter table`` to add columns in ``collection_test_data.sql`` and ``product_test_data.sql``, those files are test only instead.

Reply:
restored to the original state. added add column queries to SQL files provided